portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,33 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Sami!");
+    // Create messages array list.
```

Comment:
PTAL at guidelines for comments (pasted screenshot below). In the case of line 31, 37, 40, and maybe 45, seems to me the comments don't really add anything more than the code is saying.

<img width="1182" alt="Untitled" src="https://user-images.githubusercontent.com/2067277/83644013-a06d0280-a565-11ea-9da9-3ba5c43dc541.png">

Reply:
Thank you!